main.tex

```diff
@@ -321,14 +321,21 @@ \subsection{A Liquid Haskell primer}
         \tc{\{x:b | p\}}, which denote values of base type \tc{b} that
         satisfy predicate \tc{p}. We will write sometimes \tc{b<p>} to denote
         \tc{\{x:b | p x\}}.
-Refinements are subject to subtyping in the same way as subsets in set
-theory, so that we have
+\end{itemize}
+
+Refinement types are subject to subtyping in the same way as subsets in set
+theory. Every time we have an expression \tc{e}, the refinement
+type of \tc{e} needs to be a subtype of the refinement type expected by
+the context where \tc{e} appears. For instance, in the following function
```

Comment:
```suggestion
theory. Every time we have an expression \tc{e}, the refinement
type inferred for \tc{e} needs to be a subtype of the refinement type expected by
the context in which \tc{e} appears. For instance, in the following function
```
(where->in which isn't completely necessary, I don't think, but it feels a little better to my ear)